generative_ai/embedding_preview.py

```diff
@@ -18,13 +18,31 @@
 from vertexai.language_models import TextEmbeddingInput, TextEmbeddingModel
 
 
-def embed_text(
-    texts: List[str] = ["banana muffins? ", "banana bread? banana muffins?"],
-    task: str = "QUESTION_ANSWERING",
-    model_name: str = "text-embedding-preview-0409",
+def embed_code_query(
+    texts: List[str] = ["Retrieve a function that adds two numbers"],
```

Comment:
Can we inline all the function arguments into the code samples? It makes the samples easier to read.

Reply:
Done.